libs/wingsdk/src/shared-azure/function.inflight.ts

```diff
@@ -28,6 +28,16 @@ export class FunctionClient implements IFunctionClient {
     }
   }
 
+  /**
+   * Invokes the function asynchronously, passing the given payload as an argument.
+   * @returns immediately once the event has been handed off to AWS Lambda.
+   */
+  public async invokeAsync(payload: string): Promise<void> {
+    return Promise.reject(
+      `invokeAsync is not implemented: (payload=${payload})`
+    );
```

Comment:
```suggestion
    throw new Error("invokeAsync not implemented");
```

I think this is good enough, no?

Reply:
You must use the parameter or you won't be able to compile because of the linting warning.

Reply:
I guess we can try this
```suggestion
    payload;
    throw new Error("invokeAsync is not implemented");
```